driver/pelican/pelican.go

```diff
@@ -169,6 +221,49 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 		}
 	}
 
+	// Thermostat History Object Request to retrieve time stamps from past hour
+	timezone, timeErr := time.LoadLocation(pel.timezone)
```

Comment:
This logic will move into `DiscoverPelicans`, and now you can do something along the lines of `time.Now().In(pel.timezone)` to construct the start and end timestamps.